cmd/mcp/main.go

```diff
@@ -155,7 +155,12 @@ var serveCmd = &cobra.Command{
 }
 
 func serveCmdRun(cmd *cobra.Command, args []string) error {
-	if os.Getenv("KUBECONFIG") == "" {
+	inCluster := os.Getenv("KUBERNETES_SERVICE_HOST") != ""
+	if inCluster {
+		log.Printf("Starting in-cluster MCP server with read-only mode: %t", rootArgs.readOnly)
```

Comment:
```suggestion
		log.Printf("Starting in-cluster MCP server with read-only mode set to %t", rootArgs.readOnly)
```

The current version of this message is a bit confusing, I read this on the beginning of the review and understood that MCP in-cluster would run only in read-only mode, then I came back here and read the message again and understood that it's not the case